src/dock_gui.cpp

```diff
@@ -125,6 +125,15 @@ struct BuildDocksToolbarWindow : Window {
 		if (!can_build) {
 			DeleteWindowById(WC_BUILD_STATION, TRANSPORT_WATER);
 			DeleteWindowById(WC_BUILD_DEPOT, TRANSPORT_WATER);
+
+			/* Show in the tooltip why this button is disabled. */
+			this->GetWidget<NWidgetCore>(WID_DT_DEPOT)->SetDataTip(SPR_IMG_AIRPORT, STR_TOOLBAR_DISABLED_NO_VEHICLE_AVAILABLE);
```

Comment:
docks are not airports :)

Reply:
haha, ooopppsssss ... did it show where I copy/pasted? :D :D